src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -9156,15 +9156,6 @@ namespace SurfaceGeometry {
 						Material( TotMaterials ).WinShadeLeftOpeningMult = 0.0;
 						Material( TotMaterials ).WinShadeRightOpeningMult = 0.0;
 						Material( TotMaterials ).WinShadeAirFlowPermeability = 0.0;
-						Material( TotMaterials ).EMPDVALUE = 0.0;
-						Material( TotMaterials ).MoistACoeff = 0.0;
-						Material( TotMaterials ).MoistBCoeff = 0.0;
-						Material( TotMaterials ).MoistCCoeff = 0.0;
-						Material( TotMaterials ).MoistDCoeff = 0.0;
-						Material( TotMaterials ).EMPDaCoeff = 0.0;
-						Material( TotMaterials ).EMPDbCoeff = 0.0;
-						Material( TotMaterials ).EMPDcCoeff = 0.0;
-						Material( TotMaterials ).EMPDdCoeff = 0.0;
```

Comment:
@Myoldmopar These are initialized to zero [here](https://github.com/NREL/EnergyPlus/blob/4db1c6507021456b6879198bbdb475496a6db789/src/EnergyPlus/DataHeatBalance.hh#L875-L883).